hoot-services/src/main/java/hoot/services/utils/DbUtils.java

```diff
@@ -833,7 +834,8 @@ public static String getJobIdByTask(String taskInfo) {
         String jobId = createQuery()
                 .select(jobStatus.jobId)
                 .from(jobStatus)
-                .where(Expressions.booleanTemplate("tags->'taskInfo' like '%" + taskInfo + "%'"))
+                .where(Expressions.booleanTemplate("tags->'taskInfo' like 'taskingManager:" + taskInfo + "'"))
+                .orderBy(jobStatus.start.desc())
```

Comment:
so this handles multiple jobs (do only upload jobs get the taskInfo tag?) for the same task and only returns the most recent?

Reply:
correct since the user can run a job, it timeout, and they can run it again instead of resume it